frontend/src/components/Navigation/Navigation.vue

```diff
@@ -36,5 +36,9 @@ const routes: Route[] = [
 		title: 'Nodes',
 		path: '/nodes',
 	},
+	{
+		title: 'Tokens',
```

Comment:
Shouldn't this indicate it's CIS2 tokens?

Reply:
done